ui/desktop/src/components/MCPUIResourceRenderer.tsx

```diff
@@ -1,60 +1,61 @@
 import { UIResourceRenderer, UIActionResult } from '@mcp-ui/client';
 import { ResourceContent } from '../types/message';
-import { useState, useCallback } from 'react';
-
-// Extend UIActionResult to include size-change type
-type ExtendedUIActionResult =
-  | UIActionResult
-  | {
-      type: 'size-change';
-      payload: {
-        height: string;
-      };
-    };
+import { useCallback } from 'react';
+import { toast } from 'react-toastify';
```

Comment:
popin' some toast for user feedback, as we work to make the action handling real